### ISSUE TITLE
Not clickable in area following suggestWidget

### ISSUE THREAD

**Issue by neilsustc** [Microsoft/vscode#38484]
- VSCode Version: Code - Insiders 1.19.0-insider (91a2020698b5220e50f60682724927a810608356, 2017-11-16T05:17:48.036Z)
- OS Version: Windows_NT x64 10.0.17035
- Extensions: Extensions are disabled
---

Steps to Reproduce:

Notice the cursor shape ![image](https://user-images.githubusercontent.com/7588612/32922248-4a03210c-cb6c-11e7-96ea-efa3318d8b75.png) (can click in the editor content) and ![image](https://user-images.githubusercontent.com/7588612/32922263-57967350-cb6c-11e7-846e-509b710549fa.png) (cannot click into the editor content)

**Clickable**
![image](https://user-images.githubusercontent.com/7588612/32880611-46c0e39a-cae9-11e7-91ea-be7c04376865.png)

**Not clickable**
![image](https://user-images.githubusercontent.com/7588612/32880693-84b47d7e-cae9-11e7-8d91-801e6ece4b77.png)

**Clickable**
![image](https://user-images.githubusercontent.com/7588612/32880719-9a6c2d2e-cae9-11e7-85af-edb3091d71e7.png)

**Clickable**
![image](https://user-images.githubusercontent.com/7588612/32880798-eb69cb50-cae9-11e7-80cf-cb7f471fffee.png)

To summarize, if you click on this area, you cursor won't be placed in the corresponding line

![image](https://user-images.githubusercontent.com/7588612/32922544-ac060dbe-cb6d-11e7-8df0-fd5e487f662d.png)

**Comment by roblourens** [comment]
Please explain the problem a little more, I don't see an issue.

**Comment by neilsustc** [comment]
Okay, it kind of hard for me to express it clear. I tried. Thank you

**Comment by roblourens** [comment]
@ramya-rao-a could there be an invisible details popup there? I can't repro though.

**Comment by ramya-rao-a** [comment]
I can repro this. You need to have the docs expanded and have a suggestion list with no docs.
Thanks for reporting @neilsustc, I''l take a look

**Comment by ramya-rao-a** [comment]
This is a regression in 1.18

**Comment by Tyriar** [comment]
While clicking will hide the autocomplete box, there are still the following issues:

- The cursor is not the caret cursor
- Clicking will only hide the autocomplete box, it should also focus the position in the editor that was clicked

**Comment by ramya-rao-a** [comment]
@Tyriar Logging #43148 to track what you found.
Closing this issue as it was originally about a regression in that the said area is not letting any clicks through
